### PR TITLE
Implement icshandler class

### PR DESCRIPTION
**Pull request by kyhjonathan**
Implement base code for ics exporting. Will hold off on development until date and time have been streamlined and standardised.

### CODE REVIEW

File: src/main/java/data/Todo.java [hunk 1]
```diff
@@ -36,4 +41,30 @@ public String getTaskType () {
     public String getSaveFormat () {
         return getTaskType() + "|" + getName();
     }
+
+    public VEvent todoToVEvent(Todo todo) {
```
Comment (NicholasTanYY) [inline src/main/java/data/Todo.java:45]: Should be toDoToVEvent instead of todoToVEvent!

File: src/main/java/icshandler/IcsHandler.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        calendar.getProperties().add(new ProdId("CLINTON CALENDAR"));
+        calendar.getProperties().add(Version.VERSION_2_0);
+        calendar.getProperties().add(CalScale.GREGORIAN);
+        calendar.getComponents().add(meeting);
+
+        // Save to an ICS file
+        try (FileOutputStream fout = new FileOutputStream(filePath)) {
+            CalendarOutputter outputter = new CalendarOutputter();
+            outputter.output(calendar, fout);
+        }
```
Comment (NicholasTanYY) [inline src/main/java/icshandler/IcsHandler.java:63]: Can consider splitting these into small methods, though it is already generally quite readable.

**Review by NicholasTanYY — COMMENTED** [review]
Good job on implementing the ics class creator! We could definitely work with the dummy examples provided in a later iteration. LGTM.

File: src/main/java/data/Todo.java [hunk 3]
```diff
@@ -36,4 +41,30 @@ public String getTaskType () {
...
+        startDate.set(java.util.Calendar.SECOND, 0);
+
+        // End Date is on: April 1, 2023, 13:00
+        java.util.Calendar endDate = new GregorianCalendar();
+        endDate.set(java.util.Calendar.MONTH, java.util.Calendar.APRIL);
+        endDate.set(java.util.Calendar.DAY_OF_MONTH, 1);
+        endDate.set(java.util.Calendar.YEAR, 2023);
+        endDate.set(java.util.Calendar.HOUR_OF_DAY, 13);
+        endDate.set(java.util.Calendar.MINUTE, 0);
+        endDate.set(java.util.Calendar.SECOND, 0);
```
Comment (clarencepohh) [inline src/main/java/data/Todo.java:61]: Could consider abstracting these out

**Pull request merged**